### PR TITLE
feature/SPRIND-127

### PR DESCRIPTION
**Pull request by BtencateSphereon**

### CODE REVIEW

File: src/screens/Settings/AccountScreen/index.tsx [hunk 1]
```diff
@@ -63,29 +81,34 @@ const AccountScreen = () => {
...
+                  <ProviderDescription>
+                    <SSITextH3LightStyled>Ausweis eID</SSITextH3LightStyled>
+                    <SSITextH4LightStyled>German Bundesdruckerei</SSITextH4LightStyled>
+                  </ProviderDescription>
+                </ProviderCardRow>
+                <ImportInformationSummary data={pidInfo} />
+                <SectionTitle style={{paddingLeft: 0, marginTop: 10}}>More</SectionTitle>
+                <MoreContainer>
+                  <NavigationItem
+                    // @ts-ignore
```
Comment (sanderPostma) [inline src/screens/Settings/AccountScreen/index.tsx]: TODO comment?

File: src/screens/Settings/AccountScreen/index.tsx [hunk 2]
```diff
@@ -63,29 +81,34 @@ const AccountScreen = () => {
...
+                    onPress={() => navigation.navigate(MainRoutesEnum.AGE_DERIVED_CLAIMS, { claims: pid?.uniformVerifiableCredential?.credentialSubject.age_equal_or_over })}
+                    left={<AgeIcon width={25} height={25} />}
+                    // @ts-ignore
```
Comment (sanderPostma) [inline src/screens/Settings/AccountScreen/index.tsx]: TODO comment?

**Review by sanderPostma — APPROVED** [review]
Approved, but see if you can add comments to the // @ts-ignore so we later know why it's there and whether it needs to be fixed still

File: src/screens/Settings/AccountScreen/index.tsx (hunk 1 above)
Comment (BtencateSphereon) [reply src/screens/Settings/AccountScreen/index.tsx]: added

File: src/screens/Settings/AccountScreen/index.tsx (hunk 2 above)
Comment (BtencateSphereon) [reply src/screens/Settings/AccountScreen/index.tsx]: added

**Pull request merged**